### DISCUSSION TITLE
Comportamientos de la IA

### DISCUSSION THREAD

**Discussion by gfazzito** [Godot-en-Espanol/Node-City#8] [Ideas]
Qué opinan de que los enemigos tengan un comportamiento según su tipo?
Ejemplo: el tanque básico es drunk walk, el tanqué rápido va directo a la bandera, El tanque pesado sigue al jugador, etc.

La otra que se me ocurre es crear comportamientos y asignarlos aleatoriamente al tanque 
Ejemplo: crear comportamientos de dunk walk, objetivo bandera y objetivo jugador. Cuando spawnea un tanque, se le asigna un comportamiento elegido al azar

**Comment by emrodk** [comment]
mmm... es una idea fantástica y sigue siendo simple

**Comment by jjmontes** [comment]
+1 a la primer sugerencia. Comportamientos establecidos para cada tipo de tanque.

**Reply by emrodk** [reply]
si si, la primera sin duda. La segunda se vuelve confuso

**Comment by FROSADO** [comment]
Me gusta la idea. Le da cierto toque de inteligencia, mejor que el movimiento aleatorio. 
Es mas, añadiria un cambio de comportamiento durante la vida del enemigo, transcurridos unos minutos, cambia de estrategia.
